### PR TITLE
Add ogc-api-records service to docker composition

### PR DESCRIPTION
**Pull request by jahow**
Currently not working

Addresses #145

### CODE REVIEW

**Comment by jahow** [comment]
Requires https://github.com/georchestra/datadir/pull/279 to be merged

**Comment by fvanderbiest** [comment]
Will also require a new mapping in https://github.com/georchestra/datadir/blob/docker-master/security-proxy/targets-mapping.properties

**Comment by jeanmi151** [comment]
@jahow  I am getting a crash after ogc start : 
`docker-ogc-api-records-1                    | Caused by: org.postgresql.util.PSQLException: ERROR: relation "languages" does not exist`

**Comment by jahow** [comment]
* rebased to include latest changes in the datadir
* fixed an error with a missing healthcheck on Kibana.

The service starts up correctly provided the database was populated originally by GeoNetwork. I still get 404 on all routes from Nginx on the docker composition though... @jeanmi151

**Comment by jeanmi151** [comment]
connected with https://github.com/georchestra/datadir/pull/281

File: .gitmodules [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [submodule "config"]
 	path = config
-	url = https://github.com/georchestra/datadir.git
+	url = git@github.com:georchestra/datadir.git
 	branch = docker-master
 [submodule "resources/static"]
 	path = resources/static
-	url = https://github.com/georchestra/htdocs.git
+	url = git@github.com:georchestra/htdocs.git
```
Comment (fvanderbiest) [inline .gitmodules]: We should keep https here, as some firewalls are really tough with git.

File: docker-compose.yml [hunk 2]
```diff
@@ -300,3 +295,16 @@ services:
     volumes:
       - ./resources/kibana/kibana.yml:/usr/share/kibana/config/kibana.yml
 
+  ogc-api-records:
+    image: geonetwork/gn-cloud-ogc-api-records-service:4.2.2
+    depends_on:
+      - elasticsearch
+      - database
+    environment:
+      LANG: en_US.UTF-8
```
Comment (fvanderbiest) [inline docker-compose.yml:304]: french ?

File: docker-compose.yml [hunk 3]
```diff
@@ -300,3 +295,16 @@ services:
...
+      SERVER_SERVLET_CONTEXT_PATH: /ogc-api-records
+      SPRING_CONFIG_LOCATION: file:///etc/georchestra/geonetwork/microservices/ogc-api-records/config.yml
+      SPRING_PROFILES_ACTIVE: standalone
```
Comment (fvanderbiest) [inline docker-compose.yml:324]: `JAVA_OPTS: -Dfile.encoding=UTF-8` ? see https://github.com/georchestra/ansible/commit/55f87bcf5986de2e5dc7ed34256691a1d1d3c65a

File: .gitmodules (hunk 1 above)
Comment (jeanmi151) [reply .gitmodules]: https is okay for none auth clone but not for everything it does no works (auth, modifications, pushing ...)

**Review by jeanmi151 — APPROVED** [review]

**Comment by jahow** [comment]
still needs a rebase apparently? anyway, thanks for picking this up :)

File: docker-compose.yml [hunk 4]
```diff
@@ -300,3 +295,17 @@ services:
     volumes:
       - ./resources/kibana/kibana.yml:/usr/share/kibana/config/kibana.yml
 
+  ogc-api-records:
+    image: geonetwork/gn-cloud-ogc-api-records-service:4.2.2
```
Comment (fvanderbiest) [inline docker-compose.yml:307]: ideally: should be "latest" on master, but wondering also if this should be a `geonetwork/gn-cloud-ogc-api-records-service` or a `georchestra/gn-cloud-ogc-api-records-service`, see https://github.com/georchestra/georchestra/issues/3931

Comment (jeanmi151) [reply docker-compose.yml:307]: I can set latest, 
otherwise not idea of which image to use (geonetwork or georchestra)

Comment (jeanmi151) [reply docker-compose.yml:307]: latest tag does not exist ..

Comment (jahow) [reply docker-compose.yml:307]: I think we tested it to work with this tag; seeing how much trouble we have making this all work together, staying on a floating tag sounds unfeasible to me

Comment (jeanmi151) [reply docker-compose.yml:307]: not possible i guess : 
georchestra/gn-cloud-ogc-api-records-service:latest ---> really old (1 year ago)
geonetwork/gn-cloud-ogc-api-records-service:latest ---> do not exist

Comment (jahow) [reply docker-compose.yml:307]: (I meant 4.2.2)

**Comment by jeanmi151** [comment]
Ready to land, waiting for reviews
LGTM and tested/working
please if you merge squash the commits

**Pull request merged**